### ISSUE TITLE
Focusable eyeball feature to show/hide password in form field

### ISSUE THREAD

**Issue by yannisbarlas** [Coko-Foundation/hhmi#413]
> **Migrated from GitLab**
> **Original Author:** Grigor Malo (`@grgml`)
> **Original Link:** https://gitlab.coko.foundation/coko-org/products/hhmi/hhmi/-/issues/63

The current implementation has an eyeball feature to show/hide password, but it is not focusable via keyboard. 

Correct behavior: user should be able to focus and activate the eyeball using only the keyboard
